litellm/llms/bedrock.py

```diff
@@ -729,6 +789,9 @@ def completion(
             outputText = response_body["generations"][0]["text"]
         elif provider == "meta":
             outputText = response_body["generation"]
+        elif provider == "mistral":
+            outputText = response_body["outputs"][0]["text"]
+            model_response["finish_reason"] = response_body["outputs"][0]["stop_reason"]
```

Comment:
not a blocker to merge - but did you test this with streaming @ti3x ? 

I suspect this PR need some more work for streaming support

Reply:
@ishaan-jaff it seems to work on my end, see https://github.com/BerriAI/litellm/pull/2290#discussion_r1510017323